app/views/restaurants/_form.html.erb

```diff
@@ -38,6 +38,12 @@
     <%= f.select :cuisine_id, cuisines_to_array %><br>
   </div>
 
+  <div class="field">Current Image: <%= @restaurant.image_url.to_s.split("/").last %></div>
```

Comment:
Can you clean this up?:

`@restaurant.image_url.to_s.split("/").last`?

What's going on there?